twilight-model/src/guild/permissions.rs

```diff
@@ -65,6 +68,10 @@ bitflags! {
         ///
         /// [Guild Timeouts]: https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ
         const MODERATE_MEMBERS = 1 << 40;
+        /// Allows for viewing role subscription insights.
+        const VIEW_CREATOR_MONETIZATION_ANALYTICS = 1 << 41;
+        /// Allows for using the soundboard in a voice channel
```

Comment:
This was changed in the docs PR
```suggestion
        /// Allows for using soundboard in a voice channel
```